ts/features/bonus/siciliaVola/saga/networking/handleGetVoucherBeneficiario.ts

```diff
@@ -43,10 +46,23 @@ export function* handleGetVoucherBeneficiario(
   action: ActionType<typeof svVoucherListGet.request>
 ) {
   try {
+    const {
```

Comment:
If `/beneficiario/ricercaVoucher` fails during the scroll of the list, could generate a spam of requests. Maybe it is better add a layer of backoff to avoid this case before dispatch the networking request https://pagopa.atlassian.net/wiki/spaces/IOAPP/pages/144966052/Backoff+waiting+time (other example: https://github.com/pagopa/io-app/blob/949d69f57faf0e63c7188d80813c66df5d77c23e/ts/features/bonus/bpd/saga/networking/winning-transactions/transactionsPage.ts#L87 )

Reply:
Sure, I forgot it.
I added for both `/beneficiario/ricercaVoucher` and `/beneficiario/statiVoucher` in [8396c9f](https://github.com/pagopa/io-app/commit/8396c9f7b1510692b92ca2eb417e9705f2a93917) and in [e7a5939](https://github.com/pagopa/io-app/commit/e7a59397ee08860fb88f7487137866ea7ef2d24e)